tests/test_interface.py

```diff
@@ -127,6 +129,14 @@ def test_generate_and_write_rsa_keypair(self):
     self.assertTrue(os.path.exists(test_keypath))
     self.assertTrue(os.path.exists(test_keypath + '.pub'))
 
+    # Test for a default filepath.  If 'filepath' is not given, the key's
+    # KEYID is used as the filename.  The key is saved to the current working
+    # directory.
+    default_keypath = interface.generate_and_write_rsa_keypair(password='pw')
+    self.assertTrue(os.path.exists(default_keypath))
+    self.assertTrue(os.path.exists(default_keypath + '.pub'))
```

Comment:
Do you think it would be a good idea to `assert` that the returned `default_keypath` (basename) is equal to the generated keys keyid?  (Same goes for ecdsa and ed25519 test below)